pynest/examples/music_cont_out_proxy_example/README.md

```diff
@@ -0,0 +1,16 @@
+# Music_cont_out_proxy example
+
+## Requirements
++ MUSIC 1.1.15 or higher
++ NEST 2.12.1 or higher compiled with MPI and MUSIC
```

Comment:
Shouldn't this be NEST 2.14.0 now?

Reply:
@stinebuu thanks indeed, this is 2.14.0 now. I am going to change this.